zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/ExporterResourceProvider.java

```diff
@@ -30,7 +30,7 @@ public interface ExporterResourceProvider {
    *
    * @return A {@link Set} of {@link IndexTemplateDescriptor}
    */
-  Set<IndexTemplateDescriptor> getIndexTemplateDescriptors();
+  Collection<IndexTemplateDescriptor> getIndexTemplateDescriptors();
```

Comment:
A JavaDoc update would be nice on this and the above that we changed from Set<> to Collection<>. We missed it on the previous PR.